.pre-commit-config.yaml

```diff
@@ -60,6 +60,26 @@ repos:
     hooks:
       - id: validate_manifest
 
+  # Go hooks
+  - repo: https://github.com/TekWizely/pre-commit-golang
+    rev: v1.0.0-rc.1
+    hooks:
+      # Style Checkers
+      - id: go-lint
```

Comment:
Based on the status of https://github.com/golang/lint (archived and the note at the top of the README) supplemented with https://github.com/golang/go/issues/38968 should we remove this hook? I saw that some people switched to [golangci-lint](https://github.com/golangci/golangci-lint) (in addition or instead of the recommended alternatives in the README).

Reply:
@cisagov/team-ois

Reply:
Good point.  We should strongly consider removing it, and hopefully we can replace it with something else that continues to be maintained.

Reply:
I mainly ran into this when running this configuration locally. That was the only tool in the configuration that was neither already installed nor readily available from `brew`.

Reply:
OK, go ahead and remove it with my blessing.

Reply:
I'm ok removing golang/lint for now as well.  Someone can create another PR to propose a suitable replacement- no need to hold up progress any further for this.

Reply:
@jasonodoom